README.md

```diff
@@ -46,13 +48,21 @@ For more information on Puppet module installation see [Puppet Labs: Installing
 
 PuppetLabs provides NetDev resource support for Cisco Nexus devices with their [`puppetlabs-netdev-stdlib`](https://forge.puppet.com/puppetlabs/netdev_stdlib) module. Installing the `ciscopuppet` module automatically installs both the `ciscopuppet` and `netdev_stdlib` modules.
 
-### <a name="setup-puppet-agent">Puppet Agent<a>
+##### The `puppetlabs-resource_api` module
+
+The 'ciscopuppet' module has a dependency on the `resource_api`. The Resource API is automatically installed when you install the module. Alternatively, it can be manually installed by running `puppet module install puppetlabs-resource_api`, or by following the setup instructions in the [Resource API README](https://github.com/puppetlabs/puppetlabs-resource_api#resource_api).

```

Comment:
The Resource API (the gem(?)) is not automatically installed when you install the module (which one?).

Together with the previous paragraph on puppetlabs-netdev_stdlib this can be cleaned up. e.g. mention at the top-level puppetlabs-ciscopuppet installation that it'll pull in the dependencies described below.

> only `puppet module install` installs dependencies. Specifically `r10k` does *not*, which requires the whole faffing around here.